### DISCUSSION TITLE
Histogram CPU time in Accounting

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#6351] [Q&A] [answered]
Dear all,

I've noticed that a "Histogram CPU time" plot is available in the Accounting. However, I'm not sure how to interpret the displayed data. For instance I've selected a JobGroup containing 10 jobs and I get the following plot:

![bd73e7554f6ab3ce0f46d23f42ba1bd6](https://user-images.githubusercontent.com/6996265/187436616-99f888c4-3c3e-49aa-97cb-413803587288.png)

I don't quite understand the units in the x axis neither the number of entries, which I expected to be 10. Is it maybe due to a problem of binning?

Thank you.

**Comment by atsareg** [comment, marked as answer]
I think these are job counts per CPU seconds bin.
